desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraph.java

```diff
@@ -278,9 +284,31 @@ public void paint(Graphics g) {
                             + "So add one or more such components.";
                     imagePath = "images/window/canvas-bg-plus.png";
                 } else {
-                    title = null;
+                    title=null;
                     subTitle = null;
                     imagePath = null;
+                    
+                    try {
+                        if (!_analysisJobBuilder.isConfigured(true)) {
+                            subTitle="Job is not correctly configured";
+                            warning=true; 
+                        }
+                    } catch (Exception ex) {
+                        logger.debug("Job not correctly configured", ex);
+                        final String errorMessage;
+                        if (ex instanceof UnconfiguredConfiguredPropertyException) {
+                            UnconfiguredConfiguredPropertyException unconfiguredConfiguredPropertyException = (UnconfiguredConfiguredPropertyException) ex;
+                            ConfiguredPropertyDescriptor configuredProperty = unconfiguredConfiguredPropertyException
+                                    .getConfiguredProperty();
+                            ComponentBuilder componentBuilder = unconfiguredConfiguredPropertyException.getComponentBuilder();
+                            errorMessage = "Property '" + configuredProperty.getName() + "' in "
+                                    + LabelUtils.getLabel(componentBuilder) + " is not set!";
```

Comment:
How about "Unfortunately, the property x on the component y is required, please set it to continue"

Of course, it may be a problem when there are more components of the same type, or several problems, though the latter is not critical (there will just be a new message when they fixed the first problem)